elabapy/baseapi.py

```diff
@@ -25,7 +25,7 @@ class BaseAPI(object):
 
     token: str = ""
     endpoint: str = ""
-    proxies: dict = None
+    proxies: Optional[dict] = None
```

Comment:
Use `Optional[Dict[str, str]]` for increased specificity ;)

Reply:
Done :-)
I can't really say that Mypy is my cup of tea ;-)

Reply:
Well, for me python without type hinting is hell :p